src/main/java/ru/itis/security/filters/TokenAuthenticationFilter.java

```diff
@@ -8,6 +8,7 @@
 import java.io.IOException;
 
 public class TokenAuthenticationFilter implements Filter {
+
     private final static String AUTH_HEADER = "AUTH";
```

Comment:
It's an error. You have this "AUTH" in properties.

Reply:
removed in properties